self-core-impl/src/test/java/com/selfxdsd/core/managers/StoredProjectManagerTestCase.java

```diff
@@ -125,6 +126,140 @@ public void returnsProjects() {
         );
     }
 
+    /**
+     * PmUser returns its username.
+     */
+    @Test
+    public void pmUserReturnsUsername() {
+        final Projects assigned = Mockito.mock(Projects.class);
+        final Projects all = Mockito.mock(Projects.class);
+        Mockito.when(all.assignedTo(1)).thenReturn(assigned);
+
+        final Storage storage = Mockito.mock(Storage.class);
+        Mockito.when(storage.projects()).thenReturn(all);
+
+
+        final ProjectManager ProjectManager = new StoredProjectManager(1,
```

Comment:
@HoreaPetrila You don't need to instantiate a real ProjectManager. Just mock it using Mockito.
Also, variable names in Java should always start with a small-case letter.